src/test/java/io/airbrake/log4javabrake2/TestAsyncSender.java

```diff
@@ -1,19 +1,33 @@
 package io.airbrake.log4javabrake2;
 
-import java.util.concurrent.CompletableFuture;
+class TestAsyncSender {}
 
-import io.airbrake.javabrake.Notice;
-import io.airbrake.javabrake.AsyncSender;
+// implements AsyncSender {
+//   public Notice notice;
 
-class TestAsyncSender implements AsyncSender {
-  public Notice notice;
+//   @Override
+//   public void setErrorHost(String host) {
+//     // TODO Auto-generated method stub
+    
+//   }
 
-  @Override
-  public CompletableFuture<Notice> send(Notice notice) {
-    this.notice = notice;
-    return new CompletableFuture<Notice>();
-  }
+//   @Override
+//   public void setAPMHost(String host) {
+//     // TODO Auto-generated method stub
+    
+//   }
 
-  @Override
-  public void setHost(String host) {}
-}
+//   @Override
+//   public CompletableFuture<Notice> send(Notice notice) {
+//     // TODO Auto-generated method stub
+//     this.notice = notice;
+//     return new CompletableFuture<Notice>();
+//   }
+
+//   @Override
+//   public CompletableFuture<ApmResponse> send(String body, String path) {
+//     // TODO Auto-generated method stub
+//     return null;
+//   }
+
+// }
```

Comment:
Probably good to remove these and other comments from the file.

Reply:
ok